api/envoy/type/http/v3/path_transformation.proto

```diff
@@ -33,10 +33,10 @@ message PathTransformation {
     }
 
     // Determines if adjacent slashes are merged into one. A common use case is for a request path
-    // header. Using this option in `:ref: PathNormalizationOptions
```

Comment:
this link is currently broken - not entirely clear what it should point to as the link itself also doesnt work

updating it to point to `normalize_path` in the http_conn_man proto